test/validation/api/classification/odp_classification_tests.c

```diff
@@ -378,6 +377,169 @@ void test_cls_pmr_chain(odp_bool_t enable_pktv)
 	odp_packet_free(pkt);
 }
 
+void configure_cls_pmr_chain_rev(odp_bool_t enable_pktv)
```

Comment:
This function is almost the same as configure_cls_pmr_chain_rev(). They could be combined and/or refactored to avoid the code duplication and to make the real difference between them obvious and not lost in the clutter.

Reply:
Done in v2, although it's not unambiguously better.